### ISSUE TITLE
Questions regarding test time and evaluation methodology

### ISSUE THREAD

**Issue by AristotelisPap** [cbfinn/maml#44]
Hi Chelsea,

As far as I have understood by studying your paper, during test time, for example in the classification setting, you get N unseen classes and K instances for each one of them, you run some steps using gradient descent on these N*K examples and then you evaluate the model's ability to classify new instances in these N classes. If what I have written is correct, I have the following questions:

1) What is the objective over which you optimize during test time? Is it the loss function where the summation is only for the N*K examples?
2) During the evaluation, do you use examples from these N classes except the K ones that you have already seen?
3) During the evaluation, is the algorithm only provided with examples belonging to these N classes?

Thank you in advance for your time.
